components/playlist/browser/media_detector_component_manager.cc

```diff
@@ -79,6 +91,102 @@ MediaDetectorComponentManager::MediaDetectorComponentManager(
+       base::BindRepeating([](const GURL& url) {
+         // "https://bbcgoodfood.com/videos/foo"
+         if (url.path_piece() == "/") {
+           return false;
+         }
+
+         auto paths = SplitPaths(url);
+         return paths.size() == 2 && paths.front() == "videos";
+       })},
+      {net::SchemefulSite(GURL("https://bitchute.com")),
+       base::BindRepeating([](const GURL& url) {
+         // https://www.bitchute.com/video/XXXXXXXXXXXX/
+         if (url.path_piece() == "/") {
+           return false;
+         }
+
+         auto paths = SplitPaths(url);
+         return paths.size() == 2 && paths.front() == "video";
+       })},
+      {net::SchemefulSite(GURL("https://brighteon.com")),
+       base::BindRepeating([](const GURL& url) {
+         // GUID format
+         // https://www.brighteon.com/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+         if (url.path_piece() == "/") {
+           return false;
+         }
+
+         auto paths = SplitPaths(url);
+         if (paths.size() != 1) {
+           return false;
+         }
+
+         const auto guid_parts = SplitString(paths.front(), "-");
+         if (guid_parts.size() != 5) {
+           return false;
+         }
+
+         return guid_parts.at(0).length() == 8 &&
+                guid_parts.at(1).length() == 4 &&
+                guid_parts.at(2).length() == 4 &&
+                guid_parts.at(3).length() == 4 &&
+                guid_parts.at(4).length() == 12;
+       })},
+      {net::SchemefulSite(GURL("https://marthastewart.com")),
+       base::BindRepeating([](const GURL& url) {
+         // TODO(sko) Can't tell a page would have media from url.
+         return false;
+       })},
+      {net::SchemefulSite(GURL("https://rumble.com/")),
+       base::BindRepeating([](const GURL& url) {
+         // Starts with vXXXXXX (6 chars) and followed by title of video
+         // https://rumble.com/vXXXXXX-foo-bar-baz.html
+         if (url.path_piece() == "/") {
+           return false;
+         }
+
+         auto paths = SplitPaths(url);
+         if (paths.size() != 1) {
+           return false;
+         }
+
+         const auto media_page_parts = SplitString(paths.front(), "-");
+         return media_page_parts.size() > 1 &&
+                media_page_parts.front().size() == 7 &&
+                media_page_parts.front().at(0) == 'v';
+       })},
+      {net::SchemefulSite(GURL("https://ted.com")),
+       base::BindRepeating([](const GURL& url) {
+         // https://www.ted.com/talks/frances_s_chance_are_insect_brains_the_secret_to_great_ai
+         if (url.path_piece() == "/") {
+           return false;
+         }
+
+         auto paths = SplitPaths(url);
+         return paths.size() == 2 && paths.front() == "talks";
+       })},
+      {net::SchemefulSite(GURL("https://vimeo.com")),
+       base::BindRepeating([](const GURL& url) {
+         // https://vimeo.com/1234567  (5-9 digits)
+         return false;
+       })},
```

Comment:
Do we have this now as it always returns false?
Or checking digits part is not added yet?

Reply:
Oops, I missed. Thanks